plugins/woo-ai/package.json

```diff
@@ -101,5 +106,24 @@
 		"*.scss": [
 			"npm run lint:css:fix"
 		]
+	},
+	"wireit": {
+		"build:project:wp-scripts": {
+			"command": "wp-scripts build",
+			"clean": "if-file-deleted",
```

Comment:
Generally I've leaned towards treating `build:project` as a production build. Ideally this would also perform the uglification like `build` currently does.

Reply:
@ObliviousHarmony I'm confused about this,

when we run watch:build such as `pnpm run --filter='@woocommerce/plugin-woocommerce' watch:build`

It runs `"build": "pnpm --if-present --workspace-concurrency=Infinity --stream --filter=\"$npm_package_name...\" '/^build:project:.*$/'",`

But I don't want watch:build to be a production build.

Reply:
If you peek above @samueljseay you'll note that `watch:build` actually calls a different command:

```
"watch:build": "pnpm --if-present --workspace-concurrency=Infinity --filter=\"$npm_package_name...\" --parallel '/^watch:build:project:.*$/'
```

This deepens to the `wireit` script `"watch:build:project:wp-scripts"` right below this comment. That runs `wp-scripts start` and produces a watched dev build.